rdr_service/services/data_dictionary_updater.py

```diff
@@ -179,48 +323,67 @@ def _get_column_should_show_unique_values(self, column_definition, analyzer: Mod
         return False
 
     def _populate_questionnaire_key_tab(self, sheet: GoogleSheetsClient):
-        query = self.session.query(QuestionnaireHistory).options(joinedload(QuestionnaireHistory.concepts))
-        questionnaire_data_list = query.all()
+        update_helper = KeyTabUpdateHelper()
```

Comment:
Can you invoke this line, the same way you are doing the above args for sheet?

Reply:
Resolved offline